pkg/identity/ecdsa.go

```diff
@@ -49,3 +54,60 @@ func asn1ECDSASignature(r, s *big.Int) ([]byte, error) {
 		S: s,
 	})
 }
+
+func GetecdsaPrivateKey(f string) (*ecdsa.PrivateKey, error) {
```

Comment:
Go naming conventions usually have initialisms (like Elliptic Curve Digital Signature Algorithm) all capitalized. Since the parameter is a file name to be read instead of bytes, I wonder if better naming would be ReadECDSAPrivateKey. This fits the naming scheme used by Go standard packages to read files, and reduces the scope for confusion with any functions that get keys from different parameters.
```suggestion
func ReadECDSAPrivateKey(f string) (*ecdsa.PrivateKey, error) {
```